tillerinobot/src/main/java/tillerino/tillerinobot/lang/Spanish.java

```diff
@@ -0,0 +1,217 @@
+package tillerino.tillerinobot.lang;
+
+import java.util.List;
+import java.util.Random;
+
+import org.tillerino.osuApiModel.Mods;
+import org.tillerino.osuApiModel.OsuApiUser;
+
+import tillerino.tillerinobot.BeatmapMeta;
+import tillerino.tillerinobot.IRCBot.IRCBotUser;
+import tillerino.tillerinobot.RecommendationsManager.Recommendation;
+
+/**
+ * @author Sibchil https://osu.ppy.sh/u/3988045 https://github.com/sibchil
+ */
+public class Spanish implements Language {
+
+	@Override
+	public String unknownBeatmap() {
+		return "Lo siento, no conozco ese mapa. Es muy reciente, muy dif�cil, unranked o modo de osu! no est�ndar.";
+	}
+
+	@Override
+	public String internalException(String marker) {
+		return "Ehh... Parece que Tillerino humano estrope� mi cableado."
+		                + " Si no se da cuenta pronto, �podr�as [https://github.com/Tillerino/Tillerinobot/wiki/Contact informarle]? (c�digo de referencia "
+		                + marker + ")";
+	}
+
+	@Override
+	public String externalException(String marker) {
+		return "�Qu� est� pasando? Solo recibo disparates del servidor de osu. �Podr�as decirme que se supone que significa esto? 0011101001010000"

```

Comment:
Perhaps it's better `que deberìa decir esto?`?

(by the way, I'm trying to see if something can be improved but I don't know spanish completely, I'm just studying it a bit at school :P )